src/utils/feature-flags.ts

```diff
@@ -9,7 +9,8 @@ declare const window: any
  */
 const defaultFlagSet: LDFlagSet = {
   'alteration-ui-enabled': false,
-  'correction-ui-enabled': false
+  'correction-ui-enabled': false,
+  'change-ui-enabled': false
```

Comment:
Are any of the old flags now obsolete? (Would we ever want to disable alterations or corrections in Prod?)

Reply:
Good question, i doubt it! 
Lets have a closer look at this but my first thought is that we can remove the two older flags 👍

Reply:
On second thought, these 3 flags now control the 3 types of filings that this UI supports. Let's leave these in a little longer (in case we break anything or whatever).

Reply:
(Kind of like a maintenance page that would pre-empt an entire UI, these flags would allow us to disable filing types individually. It feels like something we might want to leave in long-term.)

Reply:
Good call, certainly wouldn't hurt!